resources/packs/core/port.go

```diff
@@ -297,17 +297,32 @@ func (p *mqlPorts) listLinux() ([]interface{}, error) {
 		}
 	}
 
+	var ports []interface{}
 	tcpPorts, err := p.parseProcNet("/proc/net/tcp", "tcp", users, getProcess)
 	if err != nil {
 		return nil, err
 	}
+	ports = append(ports, tcpPorts...)
 
 	udpPorts, err := p.parseProcNet("/proc/net/udp", "udp", users, getProcess)
 	if err != nil {
 		return nil, err
 	}
+	ports = append(ports, udpPorts...)
 
-	return append(tcpPorts, udpPorts...), nil
+	tcpPortsV6, err := p.parseProcNet("/proc/net/tcp6", "tcp", users, getProcess)
```

Comment:
Do we need additional testing to ensure the parsing works for both v4 and v6?

Reply:
Sounds like a great plan.

Reply:
https://github.com/mondoohq/cnquery/pull/564